prediction_market_agent/tools/mech/utils.py

```diff
@@ -0,0 +1,92 @@
+import os
+import tempfile
+import typing as t
+from contextlib import contextmanager
+from enum import Enum
+
+from mech_client.interact import ConfirmationType, interact
+from pydantic import BaseModel
+
+from prediction_market_agent.tools.mech.api_keys import MechAPIKeys
+from prediction_market_agent.tools.mech.mech.packages.polywrap.customs.prediction_with_research_report import (
+    prediction_with_research_report,
+)
+from prediction_market_agent.tools.mech.mech.packages.valory.customs.prediction_request import (
+    prediction_request,
+)
+from prediction_market_agent.utils import completion_str_to_json
+
+
+class MechResult(BaseModel):
+    p_yes: float
+    p_no: float
+    confidence: float
+    info_utility: float
```

Comment:
We have a similar class `OutcomePrediction` somewhere else in the code - please unify